cache/sampleStore.js

```diff
@@ -33,9 +33,10 @@ const constants = {
       'infoRange',
       'okRange',
       'writers',
+      'user', // an object
     ],
     sample: ['relatedLinks', 'user'],
-    subject: ['aspectNames', 'tags', 'relatedLinks', 'geolocation'],
+    subject: ['aspectNames', 'tags', 'relatedLinks', 'geolocation', 'user'],
```

Comment:
These are the only actual change. The rest of the changes are function rename only.